.github/workflows/ci.yml

```
@@ -71,20 +71,12 @@ jobs:
       - uses: actions/checkout@v3
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1.6'
+          version: '1'
       - run: |
           julia --project=docs -e '
             using Pkg
             Pkg.develop(PackageSpec(path=pwd()))
             Pkg.instantiate()'
-      - run: |
-          julia --color=yes --project=docs/ -e '
-            using Flux
-            # using Pkg; Pkg.activate("docs")
-            using Documenter
-            using Documenter: doctest
-            DocMeta.setdocmeta!(Flux, :DocTestSetup, :(using Flux); recursive=true)
-            doctest(Flux)'
       - run: julia --project=docs docs/make.jl
```

Will make it easier to look at the logs and identify potential errors.
```suggestion
      - run: julia --color=yes --project=docs docs/make.jl
```